### ISSUE TITLE
Not able to display gallery

### ISSUE THREAD

**Issue by Philippoes** [ImmaculatePine/hermitage#10]
Hey! 

I have installed the gem and set it up, but the gallery is not being displayed.

I am using carrierwave to store my images, and I have
specified the correct attribute names so heritage has access to both
thumbnails and full image.

In my view:
```
<%= render_gallery_for @images %>
```
where @images is an array of objects with thumbnail and full images are accessible like this: @images[i].image.thumb.url & @images[i].image.url

Hermitage configuration: 
```
Hermitage.configure :images do
  original -> item { item.image.url }
  thumbnail -> item { item.image.thumb.url }
  title -> item { item.category.name }
  list_tag :ul
  item_tag :li
  list_class 'thumbnails'
  item_class 'span4'
  link_class 'thumbnail'
  image_class nil
  each_slice nil
end
```
Viewing the gallery:
<img width="585" alt="screen shot 2017-12-17 at 00 13 56" src="https://user-images.githubusercontent.com/22058587/35911861-4f8f4a2c-0bfb-11e8-9362-3fabc6d86403.png">

Clicking an image, (not being able to click to the next image, simply goes to the image url):
![screen shot 2017-12-17 at 00 16 05](https://user-images.githubusercontent.com/22058587/35911944-87fc2be6-0bfb-11e8-8198-3ac8cba920a5.png)

What am I doing wrong?

Thanks.

**Comment by ImmaculatePine** [comment]
Hi @Philippoes,

Are you sure that you added `require hermitage` to your `application.js` and the script is initialized with no errors?

**Comment by Philippoes** [comment]
Thanks for responding @ImmaculatePine ,

Yes, hermitage is added to the application.js file.
my application.js file

```
//= require rails-ujs
//= require turbolinks
//= require_tree .
//= require jquery3
//= require popper
//= require bootstrap-sprockets
//= require hermitage
```

Also, I do get this error when clicking one of the images that are being displayed: 

```
TypeError: url.indexOf is not a function
    at jQuery.fn.init.jQuery.fn.load
```

**Comment by ImmaculatePine** [comment]
Could you reproduce it in a new empty project and share the code?

**Comment by Philippoes** [comment]
Hi, here is the [Repo Link](https://github.com/Philippoes/Hermitage_error)

When you clone it down, pls make sure to seed the database with rails db:seed to seed a few test categories. This is because I am using category name as title in the images for the gallery.

You can try and upload images on the root page, and then go to the /image route to display the gallery.

**Comment by ImmaculatePine** [comment]
@Philippoes I think I found it.

Error occurs in [loadImage()](https://github.com/ImmaculatePine/hermitage/blob/master/app/assets/javascripts/hermitage.js.coffee.erb#L555) function which uses [jQuery's load()](http://api.jquery.com/load/). The problem is that they changed its behaviour in newer releases. Old versions of jQuery fallback to `load` event if callback is the only provided argument (which is the case for `hermitage` implementation). The newest version raises error you see in your logs.

Fix should be easy - all I need is to replace `.load(complete)` with `.on('load', complete)`.

Unfortunately it won't help you... The latest version of `hermitage` (which is not released to `rubygems` yet and could be installed from master branch) doesn't support Rails 5. I added this limitation due to some problems with tests when I revised them after the Rails 5 release.

To sum it up, I can fix this, but the fix will be available for Rails < 5.0 only. You could try to rollback to the older jQuery version but I'm not sure that Rails 5 supports it.

**Comment by Philippoes** [comment]
@ImmaculatePine  Thanks! Alright, I understand. I am able to rollback to an older version of jquery. Which one would be optimal to use?

**Comment by ImmaculatePine** [comment]
@Philippoes `jquery-rails@4.0.3` should work for sure. You can check what exactly version of jQuery it provides.
